tests/vcs/VCSCommitMetadataBearTest.py

```diff
@@ -0,0 +1,112 @@
+import os
+import platform
+import shutil
+import unittest
+import unittest.mock
+from pathlib import Path
+from queue import Queue
+from tempfile import mkdtemp
+
+from bears.vcs.VCSCommitMetadataBear import (VCSCommitMetadataBear,
+                                             CommitResult, COMMIT_TYPE)
+from bears.vcs.git.GitCommitMetadataBear import GitCommitMetadataBear
+from coalib.misc.Shell import run_shell_command
+from coalib.settings.Section import Section
+from coalib.testing.BearTestHelper import generate_skip_decorator
+
+
+class FakeCommitBear(GitCommitMetadataBear):
+    @classmethod
+    def check_prerequisites(cls):
+        return True
+
+    def get_head_commit(self):
```

Comment:
The name of this method is `get_head_commit_sha()`

And for this test it should only contains `raise RuntimeError('RuntimeError from test')